src/python/src/grpc/_adapter/_c/module.c

```diff
@@ -31,21 +31,31 @@
  *
  */
 
-#ifndef _ADAPTER__SERVER_H_
```

Comment:
I'll look into `git mv`, but IIRC it's just the equivalent of `git add` + `git rm`.